src/test/java/com/github/fishio/achievements/TestAchieveEnemyKill.java

```diff
@@ -37,7 +37,10 @@ public EnemyKillObserver getKillObserver() {
 		
 		when(sppf.getPlayer()).thenReturn(pf);
 		when(sppf.playerProperty()).thenReturn(sop);
-		return new EnemyKillObserver(sppf);
+		EnemyKillObserver eko = new EnemyKillObserver(sppf);
+		eko.attachAchievement(AchievementManager.ENEMY_KILL);
```

Comment:
``` eko.attachAchievement(AchievementManager.ENEMY_KILL); ``` can be removed, it already happens in the constructor of EnemyKillObserver.

<logic